src/status_im/feature_flags.cljs

```diff
@@ -13,7 +13,7 @@
    {::wallet.bridge-token              (enabled-in-env? :FLAG_BRIDGE_TOKEN_ENABLED)
     ::wallet.edit-derivation-path      (enabled-in-env? :FLAG_EDIT_DERIVATION_PATH)
     ::wallet.remove-account            (enabled-in-env? :FLAG_REMOVE_ACCOUNT_ENABLED)
-    ::community.edit-account-selection (enabled-in-env? :FLAG_EDIT_ACCOUNT_SELECTION_ENABLED)}))
+    ::community.edit-account-selection true}))
```

Comment:
FYI: This flag will be removed and all related code before this PR merged and after QAs approve it.